client/src/components/album/Track.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const TrackWrapper = styled.li`
+  cursor: pointer;
+  list-style: none;
+  padding: 7px 10px;
+  box-sizing: border-box;
+  user-select: none;
+
+  background: ${props => (props.isPlaying ? props.theme.blackDark : null)};
+
+  &:hover {
+    background: ${props => props.theme.blackDark};
+  }
+
+  &:hover .track__note {
+    display: none;
+  }
+
+  &:hover .track__arrow {
+    display: block;
+    padding-bottom: 10px;
+  }
+
+  &:first-child {
+    margin-top: 0;
+  }
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+`;
+
+const TrackItem = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  &:hover .track-more-info {
+    display: block;
+  }
+`;
+
+const TrackLeftPart = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const TrackRightPart = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  .track-more-info {
+    display: none;
+  }
+`;
+const TrackTitle = styled.p`
+  margin: 6px 0;
+  font-size: 16px;
+  font-weight: 500;
+`;
+const AuthorName = styled.p`
+  color: ${props => props.theme.lightWhite} !important;
+  letter-spacing: 0.04rem;
+  margin-top: -9px !important;
```

Comment:
!important should be avoided if possible. You should be able to achieve the same thing by changing CSS specificity.